py/src/ai/h2o/sparkling/H2OContext.py

```diff
@@ -218,7 +200,8 @@ def asSparkFrame(self, h2oFrame, copyMetadata=True):
         if isinstance(h2oFrame, H2OFrame):
             frame_id = h2oFrame.frame_id
             jdf = self._jhc.asDataFrame(frame_id, copyMetadata)
-            df = DataFrame(jdf, self._sql_context)
+            sqlContext = SparkSession.builder.getOrCreate()._wrapped
```

Comment:
nit: The varialble is named `sqlContext` but isn't it a spark session?

Reply:
https://github.com/apache/spark/blob/bcce1b10408b5c516c936e8770a598ef415d71c3/python/pyspark/sql/session.py#L231 Should be ``SqlContext``